js/modules/loki_p2p_api.js

```diff
@@ -0,0 +1,26 @@
+// const fetch = require('node-fetch');
```

Comment:
remove?

Reply:
Ya already pushed a cheeky fix for that hehe